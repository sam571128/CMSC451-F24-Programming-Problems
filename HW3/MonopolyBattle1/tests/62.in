21 441 1000000000
5 18
13 14
21 2
10 18
8 8
20 18
5 5
15 2
20 10
17 12
3 15
1 4
16 21
4 10
20 16
6 19
4 21
12 10
9 20
17 17
2 14
17 10
2 3
15 15
9 7
16 15
6 13
5 21
17 13
5 10
8 5
8 17
9 5
20 6
17 20
19 6
11 7
10 3
20 5
13 17
7 19
11 21
9 1
11 4
10 5
7 21
12 7
9 21
1 15
11 8
12 16
5 9
2 8
1 16
14 15
9 19
19 3
1 20
4 20
11 14
9 16
8 20
8 6
2 11
8 7
10 11
14 17
11 18
5 13
6 7
16 6
2 1
18 5
18 4
3 13
4 9
3 20
11 11
1 2
16 2
6 16
4 1
18 13
13 13
12 11
21 18
8 10
7 20
21 3
14 1
3 21
5 15
16 3
21 1
12 13
7 12
15 4
7 18
3 8
7 16
16 1
18 9
4 19
15 6
6 14
18 17
10 7
2 18
11 3
13 16
16 8
18 6
12 3
21 14
9 10
20 15
13 12
7 11
19 12
14 12
15 12
9 2
10 6
15 11
12 4
9 18
15 21
15 5
20 11
7 2
7 9
6 12
21 16
11 17
8 16
21 8
7 13
18 11
17 11
12 2
4 2
14 16
21 5
20 21
4 7
4 11
10 2
9 12
5 20
1 5
20 14
7 8
1 9
2 13
14 18
6 8
1 14
7 14
13 6
14 6
5 1
3 7
11 2
7 17
19 2
19 21
18 2
2 19
11 13
21 6
17 8
17 7
10 12
16 17
9 6
3 14
19 9
8 4
15 17
3 17
3 3
20 4
17 3
1 1
3 1
17 5
13 18
9 3
9 15
10 14
20 8
1 10
15 13
2 21
6 9
2 17
8 14
1 12
5 7
19 10
9 13
5 11
18 8
13 20
19 11
7 10
20 2
2 10
6 3
18 10
12 5
21 10
20 13
12 15
14 13
4 12
14 19
4 15
6 17
3 18
13 1
20 17
10 8
3 11
12 6
5 17
19 17
6 2
4 18
3 9
21 15
6 20
6 21
4 8
11 6
8 13
7 1
12 8
21 20
9 14
2 9
17 1
19 13
8 15
8 21
17 21
2 6
14 9
14 20
15 1
20 9
13 4
8 11
8 18
2 15
11 12
9 8
12 1
4 17
17 15
18 1
19 14
13 2
16 14
4 4
2 16
15 14
11 10
18 19
14 2
14 5
20 3
18 15
5 16
16 4
19 15
12 18
3 4
5 2
13 5
6 11
1 13
7 15
1 7
11 19
14 4
2 20
5 6
21 21
19 19
8 1
16 18
21 4
2 7
6 10
8 2
3 12
12 9
11 9
16 13
2 5
8 3
16 10
10 10
12 17
21 12
15 20
19 18
20 12
21 13
10 19
1 11
14 11
5 19
13 19
18 14
13 10
1 6
19 8
13 9
14 8
13 15
17 16
10 13
13 7
13 11
8 9
11 16
3 2
10 9
19 16
18 18
18 16
15 8
14 14
10 16
4 5
3 5
6 4
10 17
20 1
21 11
17 4
7 4
15 18
21 9
12 19
2 4
21 19
20 19
10 20
12 21
12 20
19 5
7 7
17 18
16 20
17 6
17 9
18 21
19 20
19 7
11 1
2 2
1 8
4 3
6 5
3 6
14 7
9 4
16 12
17 14
21 17
16 7
20 7
7 3
15 10
8 19
5 12
16 9
15 16
13 8
3 16
7 6
5 14
5 3
13 3
9 11
18 3
19 4
4 16
18 12
11 5
4 13
20 20
10 21
10 15
18 20
7 5
10 1
1 17
4 14
15 7
1 19
12 14
16 11
5 4
21 7
15 9
17 19
1 21
11 15
14 3
16 16
14 10
6 1
15 19
10 4
15 3
18 7
9 9
14 21
6 15
6 6
17 2
1 18
2 12
19 1
8 12
13 21
16 5
6 18
3 10
5 8
16 19
11 20
12 12
1 3
3 19
4 6
9 17
