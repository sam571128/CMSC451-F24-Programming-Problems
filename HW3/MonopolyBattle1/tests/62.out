8964 8964 8964 8964 8964 8964 9002 8964 9287 8964 8964 8964 9002 8964 8964 8964 8964 8964 9143 8964 8964 